subprojects/logging/src/test/groovy/org/gradle/internal/logging/console/ConsoleFunctionalTest.groovy

```diff
@@ -55,7 +55,7 @@ class ConsoleFunctionalTest extends Specification {
         then:
         ConcurrentTestUtil.poll(1) {
             assert statusBar.display == '<---> 0% INITIALIZING [0s]'
-            assert progressArea.display == [IDLE, IDLE, IDLE, IDLE]
+            assert progressArea.display == [IDLE]
```

Comment:
:x: Let's add a test that ensures that after 2 workers spawn, they go back to `IDLE` and do not, in fact, go away or blank.